tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -1,5 +1,5 @@
 /**********************************************************************
- * Copyright (C) 2023-2024 Red Hat, Inc.
+ * Copyright (C) 2023 Red Hat, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2023-2024 Red Hat, Inc.
```

you're making changes to the file so 2023-2024 is correct